src/plugin/mod.rs

```diff
@@ -1,4 +1,4 @@
-pub use self::configuration::RapierConfiguration;
+pub use self::configuration::{RapierConfiguration, SimulationToRenderTime};
```

Comment:
Looks like `TimestepMode` is missing too.

```suggestion
pub use self::configuration::{RapierConfiguration, SimulationToRenderTime, TimestepMode};
```